psl-core/src/main/java/org/linqs/psl/application/learning/weight/bayesian/GaussianProcessPrior.java

```diff
@@ -24,6 +24,7 @@
 
     public static final int MAX_RAND_INT_VAL = 100000000;
     public static final float SMALL_VALUE = 0.4f;
+    private static final int PROVIDED_CONFIG_INDEX = 0;
```

Comment:
Constants (static final variables) should generally be public.
Why hide something that cannot be changed?